src/main/java/be/seeseemelk/mockbukkit/entity/GhastMock.java

```diff
@@ -0,0 +1,55 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.ServerMock;
+import com.google.common.base.Preconditions;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Flying;
+import org.bukkit.entity.Ghast;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+
+public class GhastMock extends MobMock implements Ghast, Flying
```

Comment:
We should have a FlyingMock this can extend from

Reply:
This would be an empty class, so is it really necessary?

Reply:
Would be nicer to have if anything ever gets added to the Flying interface, but I suppose it's not necessary.

Reply:
I added it